src/lib/components/events/event-item.svelte

```diff
@@ -21,6 +21,13 @@
       .catch(() => toast({ path, type: ToastType.Error, content: errorMessage }));
   }
 
+  function formatLocation(location?: string | null, hosted = ['Discord', 'Zoom']): string {
+    // '', null, and undefined are all TBD
+    location = location?.trim() ?? '';
+    if (!location) return 'TBD';
```

Comment:
Nit: Since `location` always asserts a typeof `string` we can check specifically to see if the string is empty. 
```suggestion
    if (location === '') return 'TBD';
```

Reply:
ooo true, good catch